### ISSUE TITLE
Integration test for combined sms functionality

### ISSUE THREAD

**Issue by Ragil** [Ragil/Communicate-Indonesia#41]
We need to test the integration of the following commands:

- harvest
- plant
- sell
- look

The following scenarios outline the behaviour

Using a user from Sumatra with a single plant
```
cmd - plant 5 potato

cmd - look Sumatra plant
Total tanam Sumatra per {day} {month} 2015
potato 5

cmd - harvest 2 potato

cmd - look Sumatra
Total Panen Sumatra per {day} {month} 2015
potato 2

cmd - harvest 1 potato
cmd - sell 1 potato

cmd - look Sumatra
Total Panen Sumatra per {day} {month} 2015
potato 3

cmd - look Sumatra plant
Total tanam Sumatra per {day} {month} 2015
potato 2

cmd - look Sumatra sell
Total jual Sumatra per {day} {month} 2015
potato 1
```

Using a user from Sumatra with multiple plants
```
cmd - plant 5 potato
cmd - plant 5 mango
cmd - plant 5 banana
cmd - plant 5 tomato
cmd - plant 5 carrot
cmd - plant 5 kiwi
cmd - plant 5 durian

cmd - look Sumatra plant
Total tanam Sumatra per {day} {month} 2015
potato 5
mango 5
banana 5
tomato 5
carrot 5
kiwi 5
durian 5

cmd - harvest 5 durian
cmd - harvest 5 kiwi

cmd - look Sumatra
Total tanam Sumatra per {day} {month} 2015
durian 5
kiwi 5

cmd - look Sumatra sell
Nothing sold
```

**Comment by girikuncoro** [comment]
Actually for the query district response, only month and year (without date) are shown to make it simple in my last commit. Is this acceptable? I think it's simpler

```
Total panen di Sumatra ({month} 2015)
```

**Comment by Ragil** [comment]
Is the data bucketed by month? Why do we need time again?

**Comment by girikuncoro** [comment]
Uh actually I was thinking to put the time to show the latest update to user. But probably we can just remove the time? It'll be much simpler for user to see

**Comment by Ragil** [comment]
We can probably live without time

**Comment by girikuncoro** [comment]
Sorry I was travelling with my dad in the past 5 days. Merged but 2 questions:
1. When user input same crops multiple times, should this be aggregated or not?
For example,
```
plant 5 potato
plant 3 potato
plant 4 potato
```
when user call `look plant Sumatra`, which is the correct output?
```
Total plant in Sumatra:
potato 5
potato 3
potato 4
```
or
```
Total plant in Sumatra:
potato 12
```

2. Should we tweak this: https://github.com/Ragil/Communicate-Indonesia/blob/master/app/api/sms/main.py#L19 (or probably other parts) so that the command is executed automatically when user sends SMS? Currently I'm executing the command manually in the combined test https://github.com/Ragil/Communicate-Indonesia/blob/master/tests/api/combined/test_combined_sms.py#L48

**Comment by Ragil** [comment]
I think we should aggregate the amount and also deduct appropriately after harvest. It should represent state of the district rather than history.

We should add all the actions into the dispatcher so it triggers automatically.

**Comment by Ragil** [comment]
> Sorry I was travelling with my dad in the past 5 days.

Np. Keep in mind that this should be a side project. It shouldn't take priority over other things in life.

**Comment by girikuncoro** [comment]
Fixed in #45
